auth/user_mgt.go

```diff
@@ -525,27 +602,39 @@ func makeExportedUser(r responseUserRecord) (*ExportedUserRecord, error) {
 		}
 	}
 
+	var providerUserInfo []*UserInfo
+	for _, u := range r.ProviderUserInfo {
+		info := &UserInfo{
+			DisplayName: u.DisplayName,
+			Email:       u.Email,
+			PhoneNumber: u.PhoneNumber,
+			PhotoURL:    u.PhotoUrl,
+			ProviderID:  u.ProviderId,
+			UID:         u.RawId,
+		}
+		providerUserInfo = append(providerUserInfo, info)
+	}
+
 	resp := &ExportedUserRecord{
 		UserRecord: &UserRecord{
 			UserInfo: &UserInfo{
 				DisplayName: r.DisplayName,
 				Email:       r.Email,
 				PhoneNumber: r.PhoneNumber,
-				PhotoURL:    r.PhotoURL,
-				ProviderID:  r.ProviderID,
```

Comment:
We should actually set ProviderID to the constant string `firebase` here. That's what our other Admin SDKs do, and it's a documented behavior. @avishalom can you make the necessary changes in a follow up PR and update the tests?